changelog.md

```diff
@@ -4,6 +4,7 @@
 
 ### Devices
 
+- BinarySensor: return `None` for `BinarySensor.counter` when context timeout is not used (and don't calculate it)
```

Comment:
```suggestion
- BinarySensor: return `None` for `BinarySensor.device_state_attributes()` when context timeout is not used (and don't calculate it)
```

Reply:
this changelog is meant for xknx. 
`device_state_attributes()` is a HA integration thing.

Reply:
👍